packages/0xcert-ethereum-generic-provider/src/tests/core/provider/sign-instance-method.test.ts

```diff
@@ -16,7 +16,7 @@ spec.before(async (stage) => {
   stage.set('provider', provider);
 });
 
-spec.only('signs a message', async (ctx) => {
+spec.test('signs a message', async (ctx) => {
```

Comment:
This is true but I still needed this to make sure everything works. Since this were just small mistakes which are subsequently relevant to this PR I would keep them here since they have no bigger impact.